code/common/grafana.q

```diff
@@ -0,0 +1,148 @@
+\d .gkdb
```

Comment:
I think maybe need to change the namespace here, e.g. 

\d .grafana

the kdb bit isn't relevant